taxi/migrations/0001_initial.py

```diff
@@ -13,54 +13,169 @@ class Migration(migrations.Migration):
     initial = True
 
     dependencies = [
-        ('auth', '0012_alter_user_first_name_max_length'),
+        ("auth", "0012_alter_user_first_name_max_length"),
```

Comment:
Why did you change this? If flake8 was arguing, then it is ok

Reply:
Oh , I just ran black. , and this changed